mediator/med_io_mod.F90

```diff
@@ -601,72 +596,64 @@ subroutine med_io_write_FB(filename, iam, FB, whead, wdata, nx, ny, nt, &
        write(tmpstr,*) subname,' dimid = ',dimid
        call ESMF_LogWrite(trim(tmpstr), ESMF_LOGMSG_INFO)
 
+       ! TODO (mvertens, 2019-03-13): below is a temporary mod to NOT write hgt
        do k = 1,nf
-          call FB_getNameN(FB, k, itemc, rc=rc)
-          if (chkerr(rc,__LINE__,u_FILE_u)) return
+          if (trim(fieldnamelist(k)) == "hgt") then
```

Comment:
Why this is here. If I remember correctly Sa_topo (or maybe I am wrong) was not matching when it is compared with the cprnc. Is this control for it?

Reply:
I don't think so. But frankly, I do not remember why it was put in. Maybe in the next PR or so I can address this question along with the other ones you raised. Can you please put in an issue for this.